src/components/gameStatus/GameStatusContainer.jsx

```diff
@@ -22,19 +22,25 @@ class GameStatusContainer extends Component {
     gameLevel: PropTypes.string,
     router: PropTypes.instanceOf(Immutable.Map),
   };
+  handleStartGame = (e) => {
+    e.preventDefault();
+    const level = e.target.name;
+    this.props.startGame(level);
+  }
   render() {
     return(
       <GameStatusWrapper gameLevel={ this.props.gameLevel }>
       <GameStatusDiv>Play again?</GameStatusDiv>
         <ButtonDiv>
-          <ButtonEasy margin={ '3px' } onClick={ () => this.props.startGame('easy') }>easy</ButtonEasy>
-          <ButtonMedium margin={ '3px' } onClick={ () => this.props.startGame('medium') }>medium</ButtonMedium>
-          <ButtonHard margin={ '3px' } onClick={ () => this.props.startGame('hard') }>hard</ButtonHard>
+          {/*<ButtonEasy name='easy' margin={ '3px' } onClick={ () => this.props.startGame('easy') }>easy</ButtonEasy>*/}
+          <ButtonEasy name='easy' margin={ '3px' } onClick={ this.handleStartGame }>easy</ButtonEasy>
+          <ButtonMedium name='medium' margin={ '3px' } onClick={ this.handleStartGame }>medium</ButtonMedium>
+          <ButtonHard name='hard' margin={ '3px' } onClick={ this.handleStartGame }>hard</ButtonHard>
```

Comment:
The margin is a prop that is passed into the styled component. The styled component uses this prop to set the margin if it is provided, otherwise it defaults to another value. The button margins need to be wider in the 'StartPage' container than in the gameStatus container.